### ISSUE TITLE
Parallelise build process

### ISSUE THREAD

**Issue by mem48** [nptscot/npt#258]
It is possible to make multiple targets run in parallel

https://books.ropensci.org/targets/performance.html#parallel-workers-and-data

It may be useful to be running local tasks while waiting for routing. But will have to manage total memory usage carefully, and avoid sending too many routing jobs to cyclestreets.

**Comment by Robinlovelace** [comment]
This is a `wontfix` I think and largely a duplicate of #123. The more relevant question is not "can we parallelise the build" but "can we speed it up", focussing on the what not the how. So closing for now, unless you can make the case that a priority to work that's distinct from #123, which shows there are various ways to speed-up the build @mem48.

**Comment by mem48** [comment]
This seems like low hanging fruit to me. The routing is the most time consuming part but takes no local resources. So would be good to send one set of routes and then be prepping another symoltaniosly. For example we could be building the school rnets while the commute routes are being generated.

**Comment by mem48** [comment]
For example, the full build I'm ruining the commute route took 15 hours. A lot of processing could have been done in that time, but currently the computer sits idle.

**Comment by mem48** [comment]
Something like this

![image](https://github.com/nptscot/npt/assets/15819577/30dd44ed-f8f9-417f-aade-654bae262c5c)

**Comment by mem48** [comment]
Based on the new national build the commute routing takes ~15 hours and the whole build takes ~20 hours. So around 5 hours (25%) can be shaved off the build time by this approach.

**Comment by Robinlovelace** [comment]
Could save time. I think we should prioritise reliability and stability before switching to focus on parallelising the build. As I say it's not about parallel vs not parallel it's about the outcome: build times. So no objection to this but yet to be convinced that it's a good use of time. 

Great work on getting the build working in #253 in any case, results looking excellent!
